### DISCUSSION TITLE
@types should be `dependencies` instead of `devDependencies`

### DISCUSSION THREAD

**Discussion by andrenarchy** [directus/directus#5837] [Feature Requests]
Currently, many type packages that are not exclusively used by the dev toolchain are listed as `devDependencies`. This means a downstream package needs to install those packages manually which is annoying and error prone. Here are some examples:

* `api/package.json`: `@types/nodemailer`, `@types/express`, ...
* `packages/drive/package.json`: `@types/fs-extra`

It would be great if those could be moved to `dependencies` instead of `devDependencies`.

**Comment by rijkvanzanten** [comment]
I think the answer to both this and your other idea (https://github.com/directus/directus/discussions/5838) could be to have a shared package that's explicitly used to share the types and utils you'd use in extensions (https://github.com/directus/directus/discussions/5837) 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
